packages/api-client/src/components/ViewLayout/ViewLayoutCollapse.vue

```diff
@@ -6,24 +6,34 @@ withDefaults(
   defineProps<{
     defaultOpen?: boolean
     itemCount?: number
+    layout?: 'client' | 'reference'
   }>(),
   {
     defaultOpen: true,
     itemCount: 0,
+    layout: 'client',
   },
 )
 </script>
 <template>
   <Disclosure
     v-slot="{ open }"
     as="div"
-    class="bg-b-2 focus-within:text-c-1 text-c-2 request-item border-b-1/2"
-    :defaultOpen="defaultOpen">
-    <div class="flex items-center">
+    class="focus-within:text-c-1 text-c-2 request-item border-b-1/2"
```

Comment:
When things get a bit crazy with if statements its a little easier to understand if you use a new component instead

Reply:
So I ripped it out then I realized we probably want the styles to match so this is the way. CVA might clean this up